types/video.js/index.d.ts

```diff
@@ -2837,6 +2837,7 @@ declare namespace videojs {
          */
         interface Event extends CustomEvent {
             [key: string]: any;
+            which: string;
```

Comment:
The docs say `which` is a `number` rather than a `string`

Reply:
embarrassing, thanks for watching out.